Library/Homebrew/download_strategy.rb

```diff
@@ -563,7 +563,7 @@ class CurlGitHubPackagesDownloadStrategy < CurlDownloadStrategy
 
   def initialize(url, name, version, **meta)
     meta ||= {}
-    meta[:header] = "Authorization: Bearer"
+    meta[:header] = ["Authorization: Bearer", "Accept: application/vnd.oci.image.index.v1+json"]
```

Comment:
@sjackman This doesn't seem quite right because this is going to be used for downloading blobs too?

Reply:
Hmm. Yes, it should be revisited in that case then to add the appropriate `Accept` header depending on the expected media type of each resource.

Reply:
Done in https://github.com/Homebrew/brew/pull/11034